org.scala-ide.sdt.core.tests/src/org/scalaide/CompilerSupportTests.scala

```diff
@@ -25,12 +28,13 @@ trait CompilerSupportTests {
   /** Can be overwritten in a subclass if desired. */
   val projectName: String = getClass().getSimpleName()
 
-  private val project: IScalaProject = SDTTestUtils.createProjectInWorkspace(projectName)
+  protected final val project: IScalaProject = SDTTestUtils.createProjectInWorkspace(projectName)
```

Comment:
Let me simply answer both questions: I made it protected because

* I use it in `CompletionTests`; note that the project is leaking out though the generated compilation units anyway, so I could have left this private, but this would have resulted in less readable code.

* I didn't make it public, because in my opinion mixin methods and values like the ones in this class should always be `protected final` since they don't belong to the public interface of classes using this traits, and because overriding them normally makes no sense.

I didn't want to go so far as to apply this reasoning to all other members of this trait, but I could if you wish. Alternatively, I can also make this value public, if you think that added protection is not worth the additional characters.